model_templates/cog.yaml

```diff
@@ -7,7 +7,7 @@ build:
   cuda: "11.8"
 
   # python version in the form '3.8' or '3.8.12'
-  python_version: "3.10"
+  python_version: "3.11"
```

Comment:
just curious -- did python 3.11 add anything of note to make any of this stuff work, or is this just an opportunistic upgrade?

Reply:
I was briefly struggling with compiling torch but it's not directly necessary for any of these changes. There's another pytorch extension I want to use that's a little clearer to build against py3.11 torch